scylla-cdc-replicator/src/replicator_consumer.rs

```diff
@@ -110,6 +167,19 @@ impl ReplicatorConsumer {
                 timestamp,
             )
             .await;
+        } else if !deleted_elements.is_empty() {
+            // V = V - {}
+            self.remove_elements_from_map(
+                column_name,
+                values_for_update,
+                deleted_elements,
+                timestamp,
+            )
+            .await;
+        } else {
+            // V = V + {}
+            self.extend_map(column_name, data, values_for_update, timestamp)
+                .await;
```

Comment:
These two cases are not mutually exclusive:
```cql
cqlsh> create table ks.t (pk int, ck int, v set<int>, primary key (pk, ck));
cqlsh> update ks.t set v = {1} where pk = 0 and ck = 0;
cqlsh> update ks.t set v = v - {1}, v = v + {2} where pk = 0 and ck = 0;
cqlsh> select * from ks.t;

 pk | ck | v
----+----+-----
  0 |  0 | {2}

```
(using sets in the example, but for maps it's the same)

Reply:
I didn't think of this. Added this in a test case.